include/odp/api/spec/proto_stats.h

```diff
@@ -27,6 +27,26 @@ extern "C" {
 
 /** ODP proto stats id */
 typedef enum odp_proto_stats_id_t {
+	/** Tx proto stats id ODP_PROTO_STATS_ID_TX_*
+	 *
+	 * - All the below modes are associated to packet stats and accounting
+	 *   on Tx of a packet via Packet IO. Packet IO pktout config
+	 *   `odp_pktout_config_opt_t::bit::proto_stats` needs to be enabled for
+	 *   the offload to work.
+	 */
```

Comment:
Since we do not have modes anymore, I would rephrase this simply as: Statistics that can be enabled in proto stats objects.